CHANGELOG.md

```diff
@@ -68,6 +68,7 @@ ___
 ### App Center Data
 
 * **[Fix]** Fix an issue where invalid characters in the document ID are accepted at creation time but causing errors while trying to read or delete the document. The characters are `#`, `\`, `/`, `?`, and all whitespaces.
+* **[Feature]** `setRemoteOperationListener` method allows to be notified of a pending operation being executed when a client device goes from offline to online.
```

Comment:
Please update the corresponding release too.

Reply:
Good catch. Updated https://github.com/microsoft/appcenter-sdk-android/releases/tag/2.1.0

Reply:
But wasn't it 2.0?

Reply:
yes it was already in 2.0, need to revert this